utils/node_local_test.py

```diff
@@ -912,6 +941,120 @@ def make_pool(daos):
 
     return get_pool_list()
 
+class posix_tests():
+    """Class for adding standalone unit tests"""
+
+    def __init__(self, server, conf, pool=None, container=None):
+        self.server = server
+        self.conf = conf
+        self.pool = pool
+        self.container = container
+        self.dfuse = None
+        self.fatal_errors = False
+
+    def needs_dfuse(function):
+        """Decorator function for starting dfuse"""
+        @functools.wraps(function)
+        def _helper(self):
+            self.dfuse = DFuse(self.server,
+                               self.conf,
+                               pool=self.pool,
+                               container=self.container)
+            self.dfuse.start(v_hint=function.__name__)
+            rc = function(self)
+            if self.dfuse.stop():
+                self.fatal_errors = True
+            return rc
+        return _helper
+
+    def fail(self):
```

Comment:
(pylint-missing-docstring) Missing method docstring
(pylint-no-self-use) Method could be a function